### PR TITLE
Fix Settings Activity Action Bar button

### PR DESCRIPTION
**Pull request by SomeshThakur**

### CODE REVIEW

File: app/src/main/java/io/aniruddh/rapiddraw/activities/SettingsActivity.java [hunk 1]
```diff
@@ -126,6 +126,7 @@ private void setupActionBar() {
         if (actionBar != null) {
             // Show the Up button in the action bar.
             actionBar.setDisplayHomeAsUpEnabled(true);
+            actionBar.setHomeButtonEnabled(true);
```
Comment (SomeshThakur) [inline app/src/main/java/io/aniruddh/rapiddraw/activities/SettingsActivity.java]: This was not needed :P

**Comment by C-Aniruddh** [comment]
Have you tested this?

**Comment by SomeshThakur** [comment]
Yeah!

**Comment by C-Aniruddh** [comment]
Merging

**Pull request merged**